medusa/tv/series.py

```diff
@@ -2146,20 +2238,16 @@ def want_episode(self, season, episode, quality, forced_search=False, download_c
 
         # if it's one of these then we want it as long as it's in our allowed initial qualities
         if ep_status == WANTED:
-            log.debug(
-                u"{id}: '{show}' {ep} status is 'WANTED'. Accepting result with quality '{new_quality}'", {
-                    'id': self.indexerid,
-                    'status': ep_status_text,
-                    'show': self.name,
-                    'ep': episode_num(season, episode),
-                    'new_quality': Quality.qualityStrings[quality],
-                }
+            should_replace, reason = (
+                True, u"Current status is 'WANTED'. Accepting result with quality '{new_quality}'".format(
```

Comment:
I'm just trying to make the method always return a tuple. And not one time a tuple other time a boolean.